src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1095,6 +1127,30 @@ struct ReplaceOptions
    const std::string replacePattern;
 };
 
+void addDirectoriesToCommand(
+   bool packageFlag, const FilePath& directoryPath, shell_utils::ShellCommand* pCmd)
+{
+   // TODO: review if switching to EscapeFilesOnly is necessay
+   *pCmd << shell_utils::EscapeFilesOnly << "--" << shell_utils::EscapeAll;
+   if (!packageFlag)
+      *pCmd << string_utils::utf8ToSystem(directoryPath.getAbsolutePath());
+   else
+   {
+      FilePath rPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/R"));
+      FilePath srcPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/src"));
+      FilePath testsPath(string_utils::utf8ToSystem(directoryPath.getAbsolutePath() + "/tests"));
+   
+      if (rPath.exists())
+         *pCmd << rPath; 
+      if (srcPath.exists())
+         *pCmd << srcPath;
+      if (testsPath.exists())
+         *pCmd << testsPath;
+      else if (!rPath.exists() && !srcPath.exists())
+         LOG_ERROR_MESSAGE("Package directories not found");
```

Comment:
I think this should just be a warning, and should include the directory in which we expected to find package folders but didn't.

Reply:
(For reference, and this should be clearer in the coding standards, you typically only log an error when an operation cannot proceed. In this case we're clearly proceeding as though nothing happened, so it's a warning.)

Reply:
done!